src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/AliasStationsMockData.java

```diff
@@ -0,0 +1,10 @@
+package org.geoserver.test;
+
+public class AliasStationsMockData extends StationsMockData {
```

Comment:
This class could be transformed into a sub-private-final-static class ``LayersNamesAliasingTests.java``, or even better: instantiated has an anonymous class. If nobody else is using it, nobody should spend time trying to udnertand why this class exists :)